project/docs/credits.md

```diff
@@ -1 +0,0 @@
-{{ credits() }}
```

Comment:
Hmm, how do you inject the credits page without this?

Reply:
https://github.com/pawamoy/copier-poetry/pull/24/commits/97cda4b8df951e45efca3ec100e21abd9c00ad95